http/vibe/http/router.d

```diff
@@ -133,6 +133,12 @@ final class URLRouter : HTTPServerRequestHandler {
 		return this;
 	}
 
+	/// ditto
+	URLRouter match(HTTPMethod method, string path, HTTPServerRequestDelegate handler)
+	{
+		return match!HTTPServerRequestDelegate(method, path, handler);
+	}
```

Comment:
Looks like the templated match function above this should already do the job just fine. So this seems like a work-around for a compiler bug to me. If so, the bug should be reported (if not known yet) and referenced in a comment here.